src/database/models/__init__.py

```diff
@@ -102,6 +102,18 @@ class LocalPlaintextConfig(BaseModel):
     user = models.ForeignKey(KhojUser, on_delete=models.CASCADE)
 
 
+class SearchModel(BaseModel):
+    class ModelType(models.TextChoices):
+        TEXT = "text"
+
+    name = models.CharField(max_length=200, default="default")
+    model_type = models.CharField(max_length=200, choices=ModelType.choices, default=ModelType.TEXT)
+    bi_encoder = models.CharField(max_length=200, default="thenlper/gte-small")
+    cross_encoder = models.CharField(
```

Comment:
Won't chat/search fail if rerank is false and this value is null?

Reply:
Valid point. Updated cross_encoder to not be nullable in DB